src/components/custom/cookie-banner.tsx

```diff
@@ -1,12 +1,25 @@
 import React from 'react'
-import PropTypes from 'prop-types'
 import styled, { keyframes } from 'styled-components'
 import { Flex } from 'components/containers'
 import { Button } from 'components/form'
 import { Text, LocalizedLinkText } from 'components/elements'
 import { Localize, localize } from 'components/localization'
 import device from 'themes/device'
 
+type CookieBannerProps = {
+    is_open?: boolean
+    onAccept?: () => void
+    onDecline?: () => void
+}
+
+type WrapperProps = {
+    is_open?: boolean
+}
+
+type StyledButtonProps = {
+    mr?: string
+}
```

Comment:
This `mr` prop is not used in styling. Can u pls double check?

Reply:
not needed, i'll remove it.